utils/ANLEra_worker_options.cfg

```diff
@@ -13,6 +13,57 @@
     [/terrain]
 #enddef
 
+#define ANL_TERRAFORM IMAGE LABEL COST SOUND SHOW_IF THEN
+[option]
+message= {MENU_IMG_TXT2 {IMAGE} ("<span color='green'>" + _ {LABEL} + "</span>") ("<span size='small'>" + _ "Cost: {COST}g" + "</span>")}
```

Comment:
Macros in translatable strings are problematic. I haven't testet what kind of problems arise, but it could be split farther such as:
```
… + _ "Cost: " + {COST} + \
# po: denotes the sign for gold. Use an upper case letter if this is done in your language that way.
_"g" + …
```

This may not work correctly with MENU_IMG_TXT(2), macros which are to be phased out anyway. I also see that you reduced it's appearance in the code from 50+. Could you replace the remaining ones by the new syntax: https://wiki.wesnoth.org/DescriptionWML#WML_Format

Reply:
Couldn’t the macro thing be fixed by adding parenthesis?

Anyway, I’ll try to fix the MENU_IMG_TXT things.

Reply:
Using macros is in practice copy-paste programming – when the preprocessor the reads the WML it does that, and there will be strings such as _"Cost: 10g", _"Cost: 12g", …

For creating the translation files the preprocessor is not used, instead a python scrip scans the files for strings and does no preprocessing. In fact it threats the files as text files rather than WML.
It will find the string _"Cost: {COST}g" instead.

The way the po translation catalogues work is that thy mag msgids (the english text) to msgstrs (the translated variant). The catalogue will know the msgid _"Cost: {COST}g", but it won't have an entry for msgids such as _"Cost: 10g", whcih the game will ask for.

Using variables in translatable strings is fine though.